README.md

```diff
@@ -503,6 +503,7 @@ Below are the configuration options with their default values:
     compatibilityJSON: 'v3', // One of: 'v1', 'v2', 'v3', 'v4
     debug: false,
     removeUnusedKeys: false,
+    removeUnusedKeysExceptions: false,
```

Comment:
To enhance flexibility, I recommend adding a `filterUnusedKeys` function rather than adding the exception object. By default, it utilizes `_.differenceWith(resStoreKeys, resScanKeys, _.isEqual)` to compare two arrays of resource keys.

```js
function filterUnusedKeys({ lng, ns, resStoreKeys, resScanKeys }) {
    return _.differenceWith(resStoreKeys, resScanKeys, _.isEqual);
}
```

If you need to include exceptions, you can make a custom function like this:
```js
{
  removeUnusedKeys: true,
  filterUnusedKeys: () => ({ lng, ns, resStoreKeys, resScanKeys }) {
      let unusedKeys = _.differenceWith(resStoreKeys, resScanKeys, _.isEqual);
    
      if (ns === 'resource') {
        const exceptionKeys = ['word', 'key.word'];
        unusedKeys = _.without(unusedKeys, exceptionKeys);
      }

      return unusedKeys;
  },
}
```

Reply:
Alternatively, we can simplify the process by passing `unusedKeys` as a parameter:

```js
function filterUnusedKeys({ lng, ns, resStoreKeys, resScanKeys, unusedKeys }) {
    return unusedKeys;
}
```

Reply:
Hi @cheton,

I noted your suggestion and replaced the object with a function.
I use the second approach because it doesn't require lodash lib.
Let me know if it's okay for you.

Reply:
I made some changes in [this commit](https://github.com/i18next/i18next-scanner/pull/262/commits/0c23fa86b8f4f422b6fcf1a28a161240c025cc4f) to enhance the original `removeUnusedKeys` option, allowing it to accept a function.